src/main/java/org/fogbowcloud/arrebol/execution/docker/DockerCommandExecutor.java

```diff
@@ -11,44 +12,49 @@
     private static final Logger LOGGER = Logger.getLogger(DockerCommandExecutor.class);
     private final WorkerDockerRequestHelper workerDockerRequestHelper;
 
+    private final long poolingPeriodTime = 300;
```

Comment:
Replace the word 'time' for the corresponding unit of measure.

Reply:
Done.